pkg/shared/ruleset/disak8sstig/option/options.go

```diff
@@ -18,6 +18,34 @@ type Option interface {
 	Validate() field.ErrorList
 }
 
+// PodSelector contains generalized options for matching entities by their attribute labels
+type PodSelector struct {
+	PodMatchLabels       map[string]string `json:"podMatchLabels" yaml:"podMatchLabels"`
+	NamespaceMatchLabels map[string]string `json:"namespaceMatchLabels" yaml:"namespaceMatchLabels"`
+}
+
+var _ Option = (*PodSelector)(nil)
+
+// Validate validates that option configurations are correctly defined
+func (e PodSelector) Validate() field.ErrorList {
+	var (
+		allErrs  field.ErrorList
+		rootPath = field.NewPath("acceptedPods")
```

Comment:
It is not necessary that the rootPath is "acceptedPods". This struct can be reused in different places. The field path should be passed as parameter. For now lets use an empty root path. We can revisit the validation of different options in another PR.
```suggestion
		rootPath = field.NewPath("")
```

Reply:
Okay, I will refactor the test cases to comply with the empty initialization.